packages/firestore/src/util/proto_byte_string.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * @license
+ * Copyright 2020 Google Inc.
```

Comment:
This should be `Copyright 2020 Google LLC`. See go/copyright.

However, see below--this file may not need to be added.

Reply:
done.